doc/guides/bluetooth/gatt-pics.rst

```diff
@@ -183,6 +186,9 @@ TSPC_GATT_4_22	True		Server: Write Long Characteristic
 TSPC_GATT_4_23	True		Server: Service Changed Characteristic (C.1)
 TSPC_GATT_4_24	False (*)	Server: Configured Broadcast (C.5)
 TSPC_GATT_4_25	False (*)	Server: Execute Write Request with empty queue (C.7)
+TSPC_GATT_4_26	False (*)	Server: Client Supported Features Characteristic
```

Comment:
@Vudentz do you know if this is supported?

Reply:
That should be set as supported, it is part of the GATT (Robust) Caching introduced in 5.1.

Reply:
FYI, TSPC_GATT_4_26 ebables:
```
GATT/SR/GAS/BV-03-C
GATT/SR/GAS/BV-03-C-LT2
```